Source/WebCore/platform/mediastream/cocoa/AudioMediaStreamTrackRendererInternalUnit.cpp

```diff
@@ -48,12 +48,17 @@
 
 namespace WebCore {
 
+namespace {
```

Comment:
But how is it, in practice, making debugging harder? IIRC every time I've set a breakpoint, it has worked for me by just using equivalent of `b LocalAudioMediaStreamTrackRendererInternalUnit`.

All compilation unit local functions, regardless of name conflict probability or not are consistently marked as static and reviews will consistently nag the names to be added as internal linkage. Based on this, I would expect consistency for any function name, regardless if it is prefixed with a class name. If it is hard to debug the types, it should be hard to debug the free functions, too.

Marking the types as internal linkage has the benefit of :
 - Theoretically stops them from being considered in some template application cases (for which I cannot remember the specifics)
 - Theoretically makes the compilation a bit simpler since the names are internal
 - Theoretically enables more inlining as the compiler knows earlier that all use-sites are in the compilation unit only

I cannot currently prove any of these, so we might as well go with the original version without the anonymous namespace. It would be great if the reviews would be more predictable, though..

Reply:
Yes, in WebKit we use `static` consistently for functions local to a source file; the unused function warning we get is one of the benefits. We typically do not use anonymous namespaces for functions or for classes local to a source file. We could start doing it but I agree that we’d have to confirm the benefits. If I am wrong about the debugging problem, then I am sorry for misleading you.